domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -249,6 +330,8 @@ class ExplorationProgressController @Inject constructor(
           solution = explorationProgress.stateGraph.computeSolutionForResult(state)
           explorationProgress.stateDeck.pushStateForSolution(state)
         } finally {
+          // mark a checkpoint in the exploration if the solution is revealed.
```

Comment:
Note comment capitalization for consistency.

Reply:
done